bundles/org.openhab.binding.dwdpollenflug/src/main/java/org/openhab/binding/dwdpollenflug/internal/handler/DWDPollenflugBridgeHandler.java

```diff
@@ -0,0 +1,217 @@
+import org.openhab.binding.dwdpollenflug.internal.config.DWDPollenflugBridgeConfiguration;
+import org.openhab.binding.dwdpollenflug.internal.dto.DWDPollenflug;
+import org.openhab.binding.dwdpollenflug.internal.dto.DWDPollenflugJSON;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link DWDPollenflugBridgeHandler} is the handler for bridge thing
+ *
+ * @author Johannes DerOetzi Ott - Initial contribution
+ */
+@NonNullByDefault
+public class DWDPollenflugBridgeHandler extends BaseBridgeHandler {
+
+    private final Logger logger = LoggerFactory.getLogger(DWDPollenflugBridgeHandler.class);
+
+    private static final String DWD_URL = "https://opendata.dwd.de/climate_environment/health/alerts/s31fg.json";
+
+    private DWDPollenflugBridgeConfiguration bridgeConfig = new DWDPollenflugBridgeConfiguration();
+
+    private @Nullable ScheduledFuture<?> pollingJob;
+
+    private @Nullable DWDPollenflug pollenflug;
+
+    private final List<DWDPollenflugRegionListener> regionListeners = new CopyOnWriteArrayList<>();
+
+    private final HttpClient client;
+
+    private final Gson gson = new Gson();
+
+    public DWDPollenflugBridgeHandler(Bridge bridge, HttpClient client) {
+        super(bridge);
+        this.client = client;
+    }
+
+    @Override
+    public void initialize() {
+        logger.debug("Initializing DWD Pollenflug bridge handler");
+        bridgeConfig = getConfigAs(DWDPollenflugBridgeConfiguration.class);
+
+        if (bridgeConfig.isValid()) {
+            onUpdate();
+        } else {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR);
+        }
+    }
+
+    private synchronized void onUpdate() {
+        if (regionListeners.isEmpty()) {
+            stopPolling();
+            updateStatus(ThingStatus.ONLINE);
+        } else {
+            startPolling();
+        }
+    }
+
+    @Override
+    public void dispose() {
+        logger.debug("Handler disposed.");
+        stopPolling();
+    }
+
+    public void startPolling() {
+        final ScheduledFuture<?> localPollingJob = this.pollingJob;
+        if (localPollingJob == null || localPollingJob.isCancelled()) {
+            logger.debug("Start polling.");
+            pollingJob = scheduler.scheduleWithFixedDelay(this::poll, INITIAL_DELAY,
+                    bridgeConfig.refresh * SECONDS_PER_MINUTE, TimeUnit.SECONDS);
+        } else if (pollenflug != null) {
+            notifyOnUpdate(pollenflug);
+        }
+    }
+
+    public void stopPolling() {
+        final ScheduledFuture<?> localPollingJob = this.pollingJob;
+        if (localPollingJob != null && !localPollingJob.isCancelled()) {
+            logger.debug("Stop polling.");
+            localPollingJob.cancel(true);
+            pollingJob = null;
+        }
+    }
+
+    public void poll() {
+        logger.debug("Polling");
+        requestRefresh().handle((pollenflug, e) -> {

```

Comment:
Instead of 'e', please give the variable a name that at least gives a hint as to what the datatype is.

Reply:
Please check my fix.